overlay/ports/taglib/portfile.cmake

```diff
@@ -0,0 +1,34 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO taglib/taglib
+    REF c8b39449c383158b4d669ee97374543f9d0733bf
```

Comment:
```suggestion
    REF v1.12
```

Reply:
For the previous versions they used the commit hash instead of the tag.